### ISSUE TITLE
Using any `autofill` options in `.sassdocrc` won't document items

### ISSUE THREAD

**Issue by indrekpaas** [SassDoc/sassdoc#399]
Using
```yml
autofill: []
```
or any combination of
```yml
autofill:
  - throw
  - content
  - require
```
will produce an "empty" output file, as if there was no items to document, and fails silently.

**Comment by valeriangalliat** [comment]
Hey! Sorry for the late reply. You need to autofill `access`, otherwise all items are marked `@access auto`, and the theme don't know how to display this.

**Comment by pascalduez** [comment]
> You need to autofill access, otherwise all items are marked @access auto, and the theme don't know how to display this.

Really ? Is that the intended behavior ? ` ¯\_(ツ)_/¯`

**Comment by valeriangalliat** [comment]
Well, `@access` was never thought to *not* be autofilled I guess.

**Comment by HugoGiraudel** [comment]
Ran through this issue today. We need to fix it, it's definitely correct. :x

**Comment by valeriangalliat** [comment]
`@access` could be `public` by default so it still works when not autofilled, but if we do this we won't be able to distinguish between an explicitely public item and an item with the default value.

So when autofilled, the following code:

```scss
/// @access public
@function _test() {}
```

would and end up to be private.

Or the theme should handle `auto` access value, but I don't like this.

Or we can return something like `Object.assign('public', { auto: true })` from the `default` function, and this way we'll be able to test from the `autofill` function if it's the default value or an explicit `public` annotation, but it's a hack.

**Comment by pascalduez** [comment]
What a clusterfuck.

How about to force `autofill` to contains `access` ?
Not particularly nice but at least there shouldn't be side effects, nor breakages ?

**Comment by valeriangalliat** [comment]
Well for sure that's better than what I was proposing!

We could hook something like this in [`ensureEnvironment`](https://github.com/SassDoc/sassdoc/blob/774abb1b41e8cba6ef4579373f8615a1f452f0bf/src/sassdoc.js#L46):

```js
if (env.autofill && !env.autofill.indexOf('access') === -1) {
  env.autofill.push('access');
}
```

**Comment by FWeinb** [comment]
I like this :+1:

**Comment by valeriangalliat** [comment]
In fact it should more be in `Environment.postProcess`. Anyway, I'm not sure about this. I recall that we have a `resolve` function at annotation level, that could convert any `auto` to `public` when not autofilled. It's the cleanest solution I have in mind. I'm implementing this.

**Comment by valeriangalliat** [comment]
Fixed in 2.1.14.
